### ISSUE TITLE
Confusing order of focusable elements (at least) on the entry form page

### ISSUE THREAD

**Issue by auge8472** [auge8472/mlf2-themes#40]
Because of the predefined order of the input elements in the entry form, implemented with `tabindex`, the focusable elements in the page header "slip" behind the form elements in the order of focussing. Those elements should be focussable as first, then the subnavigation. Then, after all these elements, the form elements have to follow. Maybe it's a good idea to implement a jump menu to reach the form elements directly from the start of the page jumping over the numerous links in the header.
